CMakeLists.txt

```diff
@@ -487,16 +487,36 @@ set(KERNELS_SOURCE_DIR ${PROJECT_SOURCE_DIR}/src/kernels)
 
 find_program(UNZIPPER lbzcat bzcat REQUIRED)
 file(MAKE_DIRECTORY ${KERNELS_BINARY_DIR})
+add_subdirectory(tools/sqlite2txt)
 
 add_custom_target(generate_kernels ALL)
 
+set(MIOPEN_USE_SQLITE_PERFDB Off CACHE BOOL "Use sqlite perfdb instead of text-based.")
+if(MIOPEN_USE_SQLITE_PERFDB)
+    set(PERFDB_SUFFIX "")
+else()
+    set(PERFDB_SUFFIX ".txt")
+endif()
+
 function(unpack_db db_bzip2_file)
     get_filename_component(__fname ${db_bzip2_file} NAME_WLE)
     add_custom_command(OUTPUT ${KERNELS_BINARY_DIR}/${__fname}
-                       COMMAND ${UNZIPPER} -k ${db_bzip2_file} > ${KERNELS_BINARY_DIR}/${__fname})
+                       COMMAND ${UNZIPPER} -ck ${db_bzip2_file} > ${KERNELS_BINARY_DIR}/${__fname})
```

Comment:
There is no need for `-c` when used with `bzcat` - it breaks Windows when used explicitly (`bzcat.exe` is a regular executable), and when missing on Linux, it does not harm (shell script over `bzip2` and `-c` automatically applied?). Is there a reason you added the `-c` option here? According to the `bzcat` man page, `-c` should be used only for the `bzip2` command.

Reply:
Removed that -c. For some reason it failed for me on windows initially, but than I have forgotten to remove it after I have fixed the initial issue. Good catch!